### PR TITLE
[NON-MODULAR] Fixes custom stuff for uniform accessories on teshis, also some taur cleanup.

### PR DESCRIPTION
**Pull request by Tastyfish**
## About The Pull Request

Note almost all of the non-modular code consists of pre-existing non-modular changes. This part of the code can't really be modularized much better than this anyway.

Fixes a couple of long-standing item-rendering regressions.

### Chickencode

A while ago, we forgot about `...species.custom_worn_icons[LOADOUT_ITEM_ACCESSORY` when that icon selection system was modernized and there was no support for it. We also lost support for `...species.offset_features[OFFSET_ACCESSORY]` at some point. Doesn't really matter when.

### Cruft

I also removed a some cruft that hasn't been used for a while, including an entire commented out file that had obsolete code.

### Some taur stuff

While perusing the related code, I saw cleaned up that taur bodytype code in human_update_icons.dm since it's mostly describing mutant_styles enums anyway, so now it'd be easier to pass other ones if we ever did that.
- It also bugged me that the same code was duplicated multiple times for determining a mob's taur mode --- and it made low level assumptions about how that was stored, so I added a new `get_taur_mode()` helper proc that just returns NONE if the mob is not a taur, doesn't have the taur accessory, or it's set to an invalid one.
- For that matter, `build_worn_icon()` was just getting TRUE or FALSE passed as the taur mode; now it gets the actual taur mode and treats it as an actual mutant style enum if you want to do something with that.

## How This Contributes To The Skyrat Roleplay Experience

This mainly fixes some regressions and code jank.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image

</details>

## Changelog

:cl:
fix: Teshari uniform accessory rendering now works as expected again.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm [hunk 1]
```diff
@@ -62,10 +62,9 @@
...
 
-		if(human_user.dna.species.mutant_bodyparts["taur"])
-			var/name = human_user.dna.species.mutant_bodyparts["taur"][MUTANT_INDEX_NAME]
-			var/datum/sprite_accessory/taur/taur = GLOB.sprite_accessories["taur"][name]
-			if(taur.taur_mode & STYLE_TAUR_SNAKE)
+		var/taur_mode = human_user.get_taur_mode()
+		if(taur_mode)
+			if(taur_mode & STYLE_TAUR_SNAKE)
 				user.allowed_turfs -= list("pawprint", "hoofprint", "clawprint")
 				user.allowed_turfs += "constrict"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm]:
```suggestion
		if(taur_mode & STYLE_TAUR_SNAKE)
			user.allowed_turfs -= list("pawprint", "hoofprint", "clawprint")
			user.allowed_turfs += "constrict"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Looks good otherwise, I'll hook us up with a test-merge just to be sure since it deals with character visuals!

File: modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm (hunk 1 above)
Comment (Tastyfish) [reply modular_skyrat/modules/emotes/code/additionalemotes/turf_emote.dm]: Oh yeah, that was left-over cruft.

**Comment by tf-4** [comment]
Has this been TM'd?

**Comment by Tastyfish** [comment]
> Has this been TM'd?

Don't think so.

**Comment by tf-4** [comment]
@GoldenAlpharex

**Review by GoldenAlpharex — APPROVED** [review]
Thanks!

**Comment by Tastyfish** [comment]
Recreated the old test images real quick

image

Also curator pocket protector not being on top of tesh's face:
image

image

**Comment by GoldenAlpharex** [comment]
Worked out fine!

**Pull request merged**